src/geckomat/utilities/ecFSEOF/run_ecFSEOF.m

```diff
@@ -1,53 +1,72 @@
-function results = run_ecFSEOF(model,rxnTarget,cSource,alphaLims,Nsteps,file1,file2)
-% 
-% run_ecFSEOF
+function results = run_ecFSEOF(ecModel,rxnTarget,cSource,alphaLims,Nsteps,file_genes,file_rxns,modelAdapter)
+%run_ecFSEOF
 %
-% 	Function that runs Flux-scanning with Enforced Objective Function
-% 	for a specified production target.
-%   
-%		model     (struct) ecModel with total protein pool constraint.
-%		rxnTarget (string) Rxn ID for the production target reaction, 
-% 				  a exchange reaction is recommended.
-%		cSource	  (string) Rxn ID for the main carbon source uptake 
-%			      reaction (make sure that the correct directionality is
-%			      indicated).
-% 		alphaLims (vector) Minimum and maximum biomass yield [gDw/mmol Csource] 
-%				  for enforced objective limits
-%       Nsteps    (integer) Number of steps for suboptimal objective 
-%                 in FSEOF
-%		file1 	  (string) opt, file name for an output .txt tab-separated 
-% 				  file for the results at the genes level
-%		file2 	  (string) opt, file name for an output .txt tab-separated 
-% 				  file for the results at the reactions level		
+% Function that runs Flux-scanning with Enforced Objective Function
+% for a specified production target.
 %
-% Usage: run_ecFSEOF(ecModel,rxnTarget,cSource,alphaLims,Nsteps)
+% Input:
+%   ecModel         an ecModel in GECKO 3 format (with ecModel.ec structure).
+%   rxnTarget       rxn ID for the production target reaction, a exchange
+%                   reaction is recommended.
+%   cSource         rxn ID for the main carbon source uptake reaction.
+%   alphaLims       vector of Minimum and maximum biomass yield [gDW/mmol Csource]
+%                   for enforced objective limits (e.g. [0.5 1]). Max value: 1.
```

Comment:
I don't understand why the maximum value should be 1?

Reply:
Alpha is scaling factor for desired suboptimal growth (which fix the LB of biomass reaction), then a value higher than 1 will result in non feasible solutions. e.g if the maximum growth is 0.41 an scaling factor of 1.1 mean fix the LB to 0.451, a value that cannot be reached.

Reply:
Ok, but then the description is somewhat confusing. I thought that these values are gDW/mmol Csource, but they are actually scaling factors of the experimental growth rate.

Reply:
Yeap, you are right I will change it